products/idn/api/patch-guide.md

```diff
@@ -0,0 +1,474 @@
+---
+id: patch-guide
+title: Patch Guide
+pagination_label: Patch Guide
+sidebar_label: Patch Guide
+sidebar_position: 8
+sidebar_class_name: patchGuide
+keywords: ['patch']
+description: Read this guide to learn how to send PATCH requests. 
```

Comment:
Could we add " to SailPoint's IdentityNow APIs." at the end? Or is that _too_ descript?

Reply:
I like that - I'll add it.